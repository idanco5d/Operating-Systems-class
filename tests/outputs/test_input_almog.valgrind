==28844== Memcheck, a memory error detector
==28844== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==28844== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==28844== Command: /mnt/c/Users/illus/os/hw/1/smash
==28844== Parent PID: 28834
==28844== 
==28844== 
==28844== FILE DESCRIPTORS: 4 open (3 std) at exit.
==28844== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_input_almog.valgrind
==28844==    <inherited from parent>
==28844== 
==28844== 
==28844== HEAP SUMMARY:
==28844==     in use at exit: 0 bytes in 0 blocks
==28844==   total heap usage: 40 allocs, 40 frees, 80,206 bytes allocated
==28844== 
==28844== All heap blocks were freed -- no leaks are possible
==28844== 
==28844== For lists of detected and suppressed errors, rerun with: -s
==28844== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
