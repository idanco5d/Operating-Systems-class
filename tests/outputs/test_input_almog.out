smash> smash: got ctrl-Z
smash: process 9210 was stopped
smash> [1] sleep 10 : 9210 0 secs (stopped)
smash> sleep 10 : 0
smash> [1] sleep 10 : 9210 0 secs
smash> sleep 10 : 9210
smash: got ctrl-C
smash: process 9210 was killed
smash> smash: sending SIGKILL signal to 0 jobs:
